docs/guides/auto-backup/workload/index.md

```diff
@@ -37,556 +39,713 @@ namespace/demo created
+You have to add the auto-backup annotations to the workload that you want to backup. The following auto-backup annotations are available:
 
-- **BackupBlueprint Name:** You have to specify the `BackupBlueprint` name that holds the template for `Repository` and `BackupConfiguration` in the following annotation:
+- **BackupBlueprint Name:** You have to specify the `BackupBlueprint` name that holds the template for `BackupConfiguration` in the following annotation:
 
 ```yaml
-stash.appscode.com/backup-blueprint: <BackupBlueprint name>
+blueprint.kubestash.com/name: <BackupBlueprint name>
 ```
 
-You can also specify multiple BackupBlueprint name separated by comma (`,`). For example:
+- **BackupBlueprint Namespace:** You have to specify the `BackupBlueprint` namespace that holds the template for `BackupConfiguration` in the following annotation:
 
 ```yaml
-stash.appscode.com/backup-blueprint: daily-gcs-backup,weekly-s3-backup
+blueprint.kubestash.com/namespace: <BackupBlueprint namespace>
 ```
 
-- **Schedule:** You can specify a schedule to backup this target through this annotation. If you don't specify this annotation, schedule from the `BackupBlueprint` will be used.
+- **Sessions:** You can specify with which sessions you want to create the `BackupConfiguration`. If you don't specify this annotation, all sessions from the `BackupBlueprint` will be used. For multiple sessions, you can provide comma (,) seperated sessions name.
 
 ```yaml
- stash.appscode.com/schedule: <Cron Expression>
+ blueprint.kubestash.com/sessions: <Sessions name>
 ```
 
-- **Target Paths:** You have to specify a list of paths that you want to backup through this annotation. Use comma (`,`) to separate multiple file paths. For example, `"/my/target/dir-1,/my/target/dir-2"`.
+- **Variables:** You can specify arbitrary number of annotations for variables. KubeStash will resolve the  variables and create `BackupConfiguration` accordingly.
 
 ```yaml
-stash.appscode.com/target-paths: "<paths to backup>"
+variables.kubestash.com/<variable-name>: <Variable value>
 ```
 
-- **Volume Mounts:** You have to also specify a list of Volumes and their MountPath and SubPath where the targeted paths are located. Use `"<volumenName>:<mountPath>:<subPath>"` format to specify the volumes. The `:<subPath>` part is optional. Use comma (`,`) to specify multiple volumes and mount path. For example, `"vol-1:/mount/path-1:sub/path-1,vol-2:/mount/path-2"`.
+Example:
 
+Assume you are using a variable, named `schedule`, in `BackupBlueprint`. By this variable we can configure different backup schedule for different target. We will add this annotation in the target.
 ```yaml
-stash.appscode.com/volume-mounts: "<volume>:<mountPath>:<subPath (optional)>"
+variables.kubestash.com/schedule: "*/5 * * * *"
 ```
 
-## Backup Deployment
+## Backup StatefulSet
 
```

Comment:
Required Annotations:

- BackupBlueprint Name: Specifies the name of the BackupBlueprint to be used for this workload's backup.

```
blueprint.kubestash.com/name: <BackupBlueprint name>
```

-  BackupBlueprint Namespace: Specifies the namespace where the BackupBlueprint resides.

```
blueprint.kubestash.com/namespace: <BackupBlueprint namespace>
```

Optional Annotations:

- Session Names: Defines which sessions from the BackupBlueprint should be used for the BackupConfiguration. If you don’t specify this annotation, all sessions from the BackupBlueprint will be used. For multiple sessions, you can provide comma (,) seperated sessions name.

```
blueprint.kubestash.com/session-names: <Session names>
```

- Variables: Allows you to customize the BackupConfiguration by providing values for variables defined within the BackupBlueprint. You can define as many annotations as needed for variables.

```
variables.kubestash.com/<variable-name>: <Variable value>
```
Example:

Assuming you are using a variable named `schedule` for a session schedule in the BackupBlueprint, you can configure different backup schedules for each target application using annotations like this:

```
variables.kubestash.com/schedule: "*/5 * * * *"
```

Reply:
I think we should change `sessions` to `session-names`